src/LuaStarSystem.cpp

```diff
@@ -542,6 +543,37 @@ static int l_starsystem_attr_faction(lua_State *l)
 	}
 }
 
+/*
+* Attribute: government
+*
+* The government type used in the system
+* (PolitGovType string constant, EARTHCOLONIAL, EARTHDEMOC, EMPIRERULE, etc.
+*
+* Availability:
+*
+*   december 2017
+*
+* Status:
+*
+*   experimental
+*/
+static int l_starsystem_attr_government(lua_State *l)
+{
+	PROFILE_SCOPED()
+	StarSystem *s = LuaObject<StarSystem>::CheckFromLua(1);
+	int j = 0;
+	for (; ENUM_PolitGovType[j].name != 0; j++)
+	{
+		if (ENUM_PolitGovType[j].value == s->GetSysPolit().govType)
+			break;
+	}
+	if (ENUM_PolitGovType[j].name == 0)
+		return 0;
+
+	lua_pushstring(l, ENUM_PolitGovType[j].name);
```

Comment:
You don't need any of the above loop, you can just do:

    lua_pushstring(l, EnumStrings::GetString("PolitGovType", s->GetSysPolit().govType));